bats/tests/helpers/images.bash

```diff
@@ -8,7 +8,7 @@
 # TODO TODO TODO
 
 # Any time you add an image here you need to re-run the mirror script!
-IMAGES=(busybox nginx python python:3.9-slim ruby tonistiigi/binfmt registry:2.8.1)
+IMAGES=(alpine busybox nginx python python:3.9-slim ruby tonistiigi/binfmt registry:2.8.1)
```

Comment:
I don't see the `alpine` image in https://github.com/orgs/rancher-sandbox/packages?tab=packages&q=bats%2F